src/Tests/Moryx.Simulation.Tests/AssemblyTestCell.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) 2023, Phoenix Contact GmbH & Co. KG
 // Licensed under the Apache License, Version 2.0
 
-using Moryx.Simulation.Examples;
+using Moryx.Resources.Benchmarking;
 
 namespace Moryx.Simulation.Tests
```

Comment:
Do you want to merge the test projects as well? 🤔

Reply:
For now I don't know what to merge it with and don't see any benefit.